src/Polly.Core/Fallback/OnFallbackArguments.cs

```diff
@@ -3,4 +3,6 @@ namespace Polly.Fallback;
 /// <summary>
 /// Represents arguments used in fallback handling scenarios.
 /// </summary>
-public record OnFallbackArguments();
+public sealed class OnFallbackArguments
```

Comment:
Compared to the others, why is this one a class rather than a struct?

Reply:
We use classes for events that do not happen on a hot path. 

As opposite to predicate arguments which are all structs.